pallets/rewards/rpc/src/lib.rs

```diff
@@ -0,0 +1,71 @@
+use std::sync::Arc;
+
+use jsonrpsee::{
+	core::{async_trait, Error as JsonRpseeError, RpcResult},
+	proc_macros::rpc,
+	types::error::{CallError, ErrorObject},
+};
+pub use pallet_rewards_runtime_api::PolkadexRewardsRuntimeApi;
+use parity_scale_codec::Codec;
+use sp_api::ProvideRuntimeApi;
+use sp_blockchain::HeaderBackend;
+use sp_runtime::{generic::BlockId, traits::Block as BlockT};
+
+const RUNTIME_ERROR: i32 = 1;
+
+#[rpc(client, server)]
```

Comment:
This crate is not compiling. On build it's crushes with error `could not find 'client' in 'core'`. Please add pallets to the root `Cargo.toml`? this is why I assume CI is not failed...

Reply:
Included into the `Workspace` and updated the `Cargo.toml` to fix the build